main.md

```diff
@@ -274,6 +274,8 @@ A number of protocols that have been standardized, or are in the process of bein
 
 - Self-Issued OpenID Provider v2 (SIOP V2): A standard that allows end-user to present self-attested or third party attested attributes when used with Opend ID for Verifiable Credential protocols. The user scans a QR code presented by the relying party to initiate the flow.
 
+Cross-device protocols should not be used for same-device scenarios. If the initiating device and authorization device is the same device protocols like OpenID Connect Core [@OpenID.Core] and OAuth 2.0 Authorization Code Grant as defined in [@RFC6749] is more appropriate. When using the OAuth 2.0 Authorization Code Grant, it is recommended to use it with Proof Key for Code Exchange (PKCE) as define in [@RFC7636].
```

Comment:
```suggestion
Cross-device protocols should not be used for same-device scenarios. If the initiating device and authorization device is the same device protocols like OpenID Connect Core [@OpenID.Core] and OAuth 2.0 Authorization Code Grant as defined in [@RFC6749] are more appropriate. When using the OAuth 2.0 Authorization Code Grant, it is recommended to use it with Proof Key for Code Exchange (PKCE) as define in [@RFC7636].
```
('is more appropriate' -> 'are more appropriate')

Possibly the final sentence should refer to https://datatracker.ietf.org/doc/html/draft-ietf-oauth-security-topics but perhaps the final sentence is also getting off topic for this document.

Reply:
@jogu Do you want to suggest alternative text for the final sentence, or should it just be deleted?

Reply:
I think I'd veer towards just deleting it.